operators/src/opencl/cl_program.rs

```diff
@@ -1270,12 +1275,12 @@ unsafe impl Sync for RasterInfo {}
 unsafe impl OclPrm for RasterInfo {}
 
 impl RasterInfo {
-    pub fn from_raster<T: Pixel>(raster: &Raster2D<T>) -> Self {
+    pub fn from_raster<T: Pixel>(raster: &GridArray2D<T>) -> Self {
```

Comment:
I guess this will be a problem if we get a GridArray here instead of a Raster because we don't have the information about the geo transform (origin, scale) which are right now not extracted but needed for computations...

Reply:
i guess we can remove the GridArray  at this point and work with RasterTile. Maybe we can then  add a to Buffer for GridArray. If this works, it could be a way to remove the macros for typed GridArray.

Reply:
i renamed GridArray to Grid.